### ISSUE TITLE
Allow url without .git suffix

### ISSUE THREAD

**Issue by mborejdo** [yoannfleurydev/gitweb#17]
**Is your feature request related to a problem? Please describe.**
using gitweb on a repo which has not been checked out with the .git-suffix results in a panic.

```
thread` 'main' panicked at 'called `Option::unwrap()` on a `None` value'
```

**Describe the solution you'd like**
The regex should allow all valid git urls.

**Comment by yoannfleurydev** [comment]
Hi, thanks a lot for opening this issue, that is a nice catch.

**Comment by yoannfleurydev** [comment]
Hi.

So I did a complete rewrite of the code as it was poorly written the first time as I did code this CLI to learn Rust. This issue is fixed in the new 0.2.0 release.
